src/main/java/org/bytedeco/javacpp/tools/Parser.java

```diff
@@ -4054,7 +4054,7 @@ void declarations(Context context, DeclarationList declList) throws ParserExcept
             declList.infoIterator = null;
             declList.spacing = null;
             do {
-                if (map != null && declList.infoIterator != null && declList.infoIterator.hasNext()) {
+                if (map != null && declList.infoIterator != null) {
```

Comment:
What about this one? Why does it remove the check? I guess we can kind of infer that it's always going to be true, but I'd rather leave it there...

Reply:
`hasNext()` is always true here, see the while clause. And at first iteration `infoIterator` is null.